libraries/ae-security/src/main/java/org/metaeffekt/core/security/cvss/CvssVector.java

```diff
@@ -377,7 +377,7 @@ public static Class<? extends CvssVector> classFromVersionName(String versionNam
      * @return the parsed vector or <code>null</code> if the vector could not be parsed
      */
     public static CvssVector parseVector(String vector) {
-        if (vector == null || StringUtils.isEmpty(CvssVector.normalizeVector(vector))) {
+        if (vector == null || StringUtils.isEmpty(CvssVector.normalizeVector(vector)) || vector.startsWith("SSVC:")) {
```

Comment:
The advantage of this approach is that it makes explicit what's not supported for parsing.

Reply:
Would this work, though? SSVC Vectors use the following format: `SSVCv2/E:N/X:S/A:N/H:M/P:D/2025-05-15T08:08:27Z/`

Which would not match with the `SSVC:` check above.

Reply:
Ah, right, I should probably change this to check for a "SSVCv2/" prefix then. But reading your comment below, I guess you'd prefer the other solution anyway?